doc/getting_started/setup_dv.md

```diff
@@ -1,10 +1,12 @@
 ---
-title: "Getting Started with Design Verification"
+title: "Design Verification Setup"
 ---
 
+_Before following this guide, make sure you've followed the [dependency installation and software build instructions]({{< relref "getting_started" >}})._
```

Comment:
Since this message is the same across many docs, maybe it would be good idea to create a [shortcode layout, like we do for tool versions](https://github.com/lowRISC/opentitan/blob/master/site/docs/layouts/shortcodes/tool_version.html) (see the Verilator [example](https://github.com/lowRISC/opentitan/blob/356c0543eec55b5b583592329ffa787612d06830/doc/ug/install_instructions/index.md?plain=1#L67))? That way you can keep these messages in sync.

Reply:
They're not quite all the same -- the software one just says "dependency installation". And in principle, some of the setups could separately instruct you to build software instead of assuming you've already done so. I'm concerned that using a shortcode layout would make it harder for someone to easily change the instructions for a specific page if needed.

Reply:
Ah got it. SG.